frame/staking/src/lib.rs

```diff
@@ -2033,11 +2046,15 @@ impl<T: Config> Module<T> {
 		);
 		let validator_staking_payout = validator_exposure_part * validator_leftover_payout;
 
+		// Track the number of payouts in order to track the actual weight.
+		let mut payout_count: u32 = 0;
+
 		// We can now make total validator payout:
 		if let Some(imbalance) = Self::make_payout(
 			&ledger.stash,
 			validator_staking_payout + validator_commission_payout
 		) {
+			payout_count += 1;
```

Comment:
Shouldn't increase the payout_count here.

We always assume the validator is paid to. The value in the benchmarks should only track the nominators that are paid to.

Reply:
> We always assume the validator is paid to. 
Where do we assume that?

> The value in the benchmarks should only track the nominators that are paid to.
Why not track the payout op to the validator in the benchmarks?

Reply:
ah wait i see your other comment